spacy/__init__.py

```diff
@@ -32,6 +32,7 @@ def load(
     enable: Union[str, Iterable[str]] = util._DEFAULT_EMPTY_PIPES,
     exclude: Union[str, Iterable[str]] = util._DEFAULT_EMPTY_PIPES,
     config: Union[Dict[str, Any], Config] = util.SimpleFrozenDict(),
+    pipe_instances: Dict[str, Any] = util.SimpleFrozenDict(),
```

Comment:
```suggestion
    pipe_instances: Dict[str, PipeCallable] = util.SimpleFrozenDict(),
```
cf the type we're using in `add_pipe_instance()`

Reply:
Same comment for the other methods, too.